packages/serverless/test/awslambda.test.ts

```diff
@@ -181,7 +179,7 @@ describe('AWSLambda', () => {
 
   describe('wrapHandler() on sync handler', () => {
     test('successful execution', async () => {
-      expect.assertions(10);
+      expect.assertions(9);
```

Comment:
Why did the number of expected assertions decrease?

Reply:
Because of the removal of the `expect` on line 49, which happened because that info is now set by the node client, not the aws integration.